javascript-if/main.js

```diff
@@ -0,0 +1,61 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(num) {
+  return num < 5;
+}
+
+function isEven(num) {
+  return (num % 2 === 0);
+}
+
+function startsWithJ(string) {
+  return string[0] === 'J' || string[0] === 'j';
+}
+
+function isOldEnoughToDrink(person) {
+  return person.age >= 21;
+}
+function isOldEnoughToDrive(person) {
+  return person.age >= 16;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return 'false - this is illegal';
+}
```

Comment:
The `return` value should have a `Boolean` data type:

---

![image](https://user-images.githubusercontent.com/16143771/152251121-29b67781-4d35-4f64-885d-1fe41aa11c1c.png)